ios/RNKiwiMobile/RNKiwiSharedBridge.m

```diff
@@ -0,0 +1,55 @@
+#import "RNKiwiSharedBridge.h"
+#import "RNKiwiOptions.h"
+#import <React/RCTBridge.h>
+#import <React/RCTRootView.h>
+
+@interface RNKiwiSharedBridge()
+
+@property (nonatomic, strong) NSMutableDictionary<NSString*,RCTBridge*> *bridges;
+
+@end
+
+@implementation RNKiwiSharedBridge
+
++ (id)sharedInstance {
+  static dispatch_once_t pred = 0;
+  __strong static id _sharedObject = nil;
+  dispatch_once(&pred, ^{
+    _sharedObject = [[self alloc] init];
+  });
+  return _sharedObject;
+}
+
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    _bridges = [NSMutableDictionary new];
+  }
+  return self;
+}
+
+- (NSURL *)resolveBundleURL:(id<RNKiwiOptions>)options {
+  return [options respondsToSelector:@selector(jsCodeLocation)]
+    ? [options jsCodeLocation]
+    : [[NSBundle bundleForClass:[self class]] URLForResource:@"hotels" withExtension:@"jsbundle"];
```

Comment:
Mmm maybe it has more sense to make this mandatory (to respond to selector) and do not have `hotels` as default? WDYT?

Reply:
Yeah, I think we need to expose few constants from this module so that users can just write `jsCodeLocation = [RNKiwiOptions hotelsBundle]`.